tests/queries/shell_config.sh

```diff
@@ -27,7 +27,10 @@ export CLICKHOUSE_TEST_UNIQUE_NAME="${CLICKHOUSE_TEST_NAME}_${CLICKHOUSE_DATABAS
 export CLICKHOUSE_BINARY=${CLICKHOUSE_BINARY:="$(command -v clickhouse)"}
 # client
 [ -x "$CLICKHOUSE_BINARY-client" ] && CLICKHOUSE_CLIENT_BINARY=${CLICKHOUSE_CLIENT_BINARY:=$CLICKHOUSE_BINARY-client}
-[ -x "$CLICKHOUSE_BINARY" ] && CLICKHOUSE_CLIENT_BINARY=${CLICKHOUSE_CLIENT_BINARY:=$CLICKHOUSE_BINARY client}
+if [ -x "$CLICKHOUSE_BINARY" ]; then
```

Comment:
This is wrong. `benchmark` should be treated separately (in its own block) and in a similar way as client/local/server...

Reply:
Good point, thanks! Fixed

Reply:
Nice, thank you!